src/time/span.cr

```diff
@@ -286,6 +286,10 @@ struct Time::Span
   def self.zero
     new(0)
   end
+              
+  def zero?
+    self == typeof(self).zero
```

Comment:
I don't see how `Time::Span` is a general tyThere's `Time::Span::Zero` constant more suitable for this anyhow.